src/renderer/src/components/Select.tsx

```diff
@@ -55,9 +59,9 @@ const Root = ({ label, onSelect, disabled, children, title }: TProps) => {
 
       <RadixSelect.Portal>
         <RadixSelect.Content
-          className="bg-gray-800 w-[11.625rem] min-w-[11.625rem] rounded-b shadow-xl"
+          className={`bg-gray-800 min-w-[11.625rem] rounded-b shadow-xl ${radixContextClassName || ''}`}
```

Comment:
Use StyleHelper.mergeStyles
```suggestion
          className={StyleHelper.mergeStyles("bg-gray-800 min-w-[11.625rem] rounded-b shadow-xl", radixContextClassName)}
```